crates/ra_hir/src/semantics.rs

```diff
@@ -246,6 +246,11 @@ impl<'db, DB: HirDatabase> Semantics<'db, DB> {
         self.analyze(path.syntax()).resolve_path(self.db, path)
     }
 
+    pub fn lower_path(&self, path: &ast::Path) -> Option<Path> {
```

Comment:
@flodiebold I move the lower_pat to `Semantic` , and the name of this function is ready for bikeshedding.  :)